.evergreen/run-tests.sh

```diff
@@ -111,6 +111,30 @@ if [ "${SKIP_CRYPT_SHARED_LIB_DOWNLOAD}" != "true" ]; then
   echo "CRYPT_SHARED_LIB_PATH=$CRYPT_SHARED_LIB_PATH"
 fi
 
+# Ensure mock KMS servers are running before starting tests.
+if [ ! "$SUPRESS_CLIENT_SIDE_ENCRYPTION" = "true" ]; then
+   await_server() {
+      for i in $(seq 300); do
+         # Exit code 7: "Failed to connect to host".
+         if curl -s "localhost:$2"; test $? -ne 7; then
+            return 0
+         else
+            sleep 1
+         fi
+      done
+      echo "could not detect '$1' server on port $2"
+   }
+   # * List servers to await here ...
+   await_server "KMS", 5698
+
+  echo "finished awaiting servers"
+   if ! test -d /cygdrive/c; then
+      # We have trouble with this test on Windows. only set cryptSharedLibPath on other platforms
```

Comment:
```suggestion
      # We have trouble with this test on Windows, only set cryptSharedLibPath on other platforms.
```
I assume most of this is copied from C? Why do we have trouble on Windows with these new unified tests?

Reply:
I believe this is because you cannot use a non-system install of libmongocrypt on windows.  In particular, the Go driver expects libmongocrypt to be in the path: C:\libmongocrypt.